bool finish
---
float64 ud_neckangle